src/app/body/milestone-details/milestone-details.component.ts

```diff
@@ -60,14 +61,15 @@ export class MilestoneDetailsComponent implements OnInit {
 
   ngOnInit(): void {
     this.milestoneId = this.route.snapshot.params['MilestoneId'];
-    this.navbarHandler.addToNavbar(this.milestoneId);
+    this.navbarHandler.addToNavbar("Milestones / "+ this.milestoneId);
```

Comment:
Navbar can be handled in a better way. Just give milestone id here, and reset of the navbar part can be added from parent component